ui/apps/platform/src/Containers/AccessControl/AccessScopes/EffectiveAccessScopeTable.tsx

```diff
@@ -243,10 +243,16 @@ function EffectiveAccessScopeTable({
             <Flex className="pf-u-pt-sm pf-u-pb-sm pf-u-pl-lg">
                 <Flex spaceItems={{ default: 'spaceItemsSm' }} className="pf-u-pb-sm">
                     <FlexItem>
-                        <span className="pf-u-font-size-sm pf-u-text-nowrap">Cluster filter:</span>
+                        <label
+                            htmlFor="Cluster_filter"
+                            className="pf-u-font-size-sm pf-u-text-nowrap"
+                        >
+                            Cluster filter:
+                        </label>
                     </FlexItem>
                     <FlexItem>
                         <TextInput
+                            id="Cluster_filter"
```

Comment:
This is fine. Just weighing in on the style of the IDs, like you asked.

The translation from camelCase to `id` I've most commonly seen is with hyphens instead of underscores. It mimics the common practice of hyphens in CSS class names, and it seems to be what HTML/CSS designers use for multi-word attributes in general. (See, for instance, the `ata-ouia-component-id="OUIA-Generated-TextInputBase-3"` in the markup you quoted.)

Having said that, I don't have a strong opinion between uderscores vs. hyphens. I agree camelCase is harder to read.

Reply:
Aha, that reminds me that in context of technical documentation, hyphen was often a meaningful character.

Reply:
Google search crawler interprets hyphens as spaces for SEO-friendly URLs, but ignored underscores, at least a few years ago. Not directly relevant, just a similar pattern.

Also, I remember that Angular ecosystem considered camelCase and kebab-case as equivalent.
https://indepth.dev/posts/1181/angular-cli-camelcase-or-kebab-case-2